libs/libwebsockets/patches/100-fix-uninitialized-variable-usage.patch

```diff
@@ -0,0 +1,22 @@
+From 3589cc48e28cd06c4143da244af5ea91b06f2e35 Mon Sep 17 00:00:00 2001
```

Comment:
This commit was changed since you posted it here.

Can you please replace it by this one:
https://github.com/warmcat/libwebsockets/commit/bfce9f86187b0846c3eab359591e64ed4fa4bb0a.patch ?

Reply:
@BKPepe updated.